rgd/geodata/static/geodata/css/sass/_base.scss

```diff
@@ -11,6 +11,7 @@ html {
   font-weight: 300;
   height: 100%;
   margin: 0;
+  overflow: hidden;
```

Comment:
@banesullivan - I'm not sure what the little scroll issue is. I've gone through every element in dev tools and made sure the padding and margins were set properly and that there was no additional element or pseudo element causing the page to be slightly higher.

This is a bit of a hacky solution, but it works. Let me know if you have any issues with it.

Reply:
Huh. very odd. For now, this works for me